static/app/views/issueDetails/streamline/foldSection.tsx

```diff
@@ -91,18 +43,37 @@ export const FoldSection = forwardRef<HTMLElement, FoldSectionProps>(function Fo
     children,
     title,
     actions,
-    sectionKey,
+    config,
     initialCollapse = false,
     preventCollapse = false,
     ...props
   },
   ref
 ) {
   const organization = useOrganization();
-  const [isCollapsed, setIsCollapsed] = useLocalStorageState(
+  const {sectionData, dispatch} = useEventDetails();
+  const {key: sectionKey} = config;
+  // Does not control open/close state. Controls what state is persisted to local storage
+  const [isInitiallyCollapsed, setIsInitiallyCollapsed] = useLocalStorageState(
     `${LOCAL_STORAGE_PREFIX}${sectionKey}`,
     initialCollapse
   );
+  const isCollapsed = !sectionData[sectionKey]?.isOpen;
+
+  // On initial load we want to reflect the local storage state.
+  // Afterward, it is written to, but ignored.
+  useEffect(() => {
+    dispatch({
+      type: 'UPDATE_SECTION_CONFIG',
+      key: sectionKey,
+      config: {
+        ...config,
+        isOpen: isInitiallyCollapsed,
+      },
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
might be able to get away without a useeffect here since this is also where it gets registered?
```suggestion
 if (!sectionData.hasOwnProperty(sectionKey)) {
    dispatch({
      type: 'UPDATE_SECTION_CONFIG',
      key: sectionKey,
      config: {
        ...config,
        isOpen: isInitiallyCollapsed,
      },
    });
}
```